frontend/lib/swr-cache/index.ts

```diff
@@ -111,6 +111,12 @@ export const withCache = <
          req.write(postData);
          req.end(() => {
             const elapsed = performance.now() - start;
+            logger.info('revalidation request end', {
+               path: req.path,
+               method: req.method,
```

Comment:
This is always going to be 'POST' right?

Reply:
Yes. I should probably remove it then.

Reply:
Up to you, I just wanted to make sure I understood correctly